README copy.md

```diff
@@ -0,0 +1,25 @@
+# provider-kops
```

Comment:
do you want to add some useful content, e.g. a quick start that tells users how to use this repo/provider into a `README.md`? That could give a lot of useful context to help people be successful on their first try with this project.